api/graphql/mutations/sequencing_group.py

```diff
@@ -36,3 +60,45 @@ async def add_comment(
             content=content,
         )
         return GraphQLComment.from_internal(result)
+
+    @strawberry.mutation
+    async def update_sequencing_group(
+        self,
+        sequencing_group: SequencingGroupUpsertInput,
+        info: Info,
+        root: 'ProjectMutations',
+    ) -> Annotated['GraphQLSequencingGroup', strawberry.lazy('api.graphql.schema')]:
+        """Update the meta fields of a sequencing group"""
+        from api.graphql.schema import GraphQLSequencingGroup
+
+        connection: Connection = info.context['connection']
+
+        # Should be moved to the sequencing group layer
+        connection.check_access_to_projects_for_ids(
+            [root.project_id], FullWriteAccessRoles
+        )
+
+        if not sequencing_group.id:
+            raise ValueError('Sequencing group ID must be provided for update')
+
+        # TODO: Review this against the route endpoint
```

Comment:
Does this TODO still need doing?

Reply:
Just reviewed it, I probably should have left more notes in the TODO to understand why I was supposed to review it ;)